db/migrate/20211011125435_add_indices_to_notes.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndicesToNotes < ActiveRecord::Migration[6.1]
+  def change
+    add_index :notes, :host_id
+    add_index :notes, :service_id
```

Comment:
I don't know what Pro's best practices are, but for production apps I've always used this approach https://thoughtbot.com/blog/how-to-create-postgres-indexes-concurrently-in

We may also be able to have an index on two columns for our db_import scenario
https://apidock.com/rails/v2.2.1/ActiveRecord/ConnectionAdapters/SchemaStatements/add_index

Reply:
A single index against two columns for our specific use case, but not strongly opinionated - postgres is pretty good at working things out